iscsi/initiator.go

```diff
@@ -29,46 +30,40 @@ const (
 	scanModeManual = "manual"
 	scanModeAuto   = "auto"
 	ScanTimeout    = 10 * time.Second
-
-	shellBinary = "sh"
 )
 
-func CheckForInitiatorExistence(ne *util.NamespaceExecutor) error {
+func CheckForInitiatorExistence(nsexec *lhns.Executor) error {
 	opts := []string{
 		"--version",
 	}
-	_, err := ne.Execute(iscsiBinary, opts)
+	_, err := nsexec.Execute(iscsiBinary, opts, lhtypes.ExecuteDefaultTimeout)
 	return err
 }
 
-func UpdateScsiDeviceTimeout(devName string, timeout int64, ne *util.NamespaceExecutor) error {
-	opts := []string{
-		"-c",
-		fmt.Sprintf("echo %v > /sys/block/%v/device/timeout", timeout, devName),
-	}
-	_, err := ne.Execute(shellBinary, opts)
-	return err
+func UpdateScsiDeviceTimeout(devName string, timeout int64, nsexec *lhns.Executor) error {
+	deviceTimeoutFile := filepath.Join("/sys/block", devName, "device", "timeout")
+	return lhns.WriteFile(lhtypes.ProcessNone, deviceTimeoutFile, fmt.Sprint(timeout))
 }
```

Comment:
What's `lhtypes.ProcessNone`? Why not just `lhns.WriteFile(deviceTimeoutFile, fmt.Sprint(timeout))`?

Reply:
Initially, I thought there might be situations where Longhorn needed to switch to another namespace for function runs. In such cases, `lhtypes.ProcessNone` was to indicate that there was no specific process namespace to enter, and the host namespace would be used instead.

However, after re-examining the code, I realized that there are currently no scenarios that require this behavior. So, I have removed related arguments.